docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,46 @@
 # Contributing to Endless Sky
 
-## Posting issues
+## Issues
+
+### Posting issues
 
 The [issues page](https://github.com/endless-sky/endless-sky/issues) on GitHub is for tracking bugs and feature requests. When posting a new issue, please:
 
-* Check to make sure it's not a duplicate of an existing issue.
+* Check to make sure it is not a duplicate of an existing issue.
 * Create a separate "issue" for each bug you are reporting and each feature you are requesting.
-* Do not use the issues page for things other than bug reports and feature requests. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead.
+* Choose the correct issue template and fill it out to the best of your ability.
+* Do not use the issues page for things other than what match the provided issue templates. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead. Examples of what the discussion pages would be used for include topics that are not looking for a resolution (as the point of an issue is to have it be resolved) or topics that are extremely broad and not befitting of an issue because there is no single resolution to the topic.
+
+If requesting a new feature, first ask yourself: will this make the game more fun or interesting within the existing scope of the game? Remember that this is not a simulator game; while you may find such games fun or interesting, that is not the type of game that Endless Sky is trying to be.
```

Comment:
seems oddly specific to mention simulators here? There are a lot of other genres that also do not fit the bill. 

Edit: I see that it was on the old doc

Reply:
I want to echo that I found the simulator statement a bit jarring in the existing doc and still find it a bit odd here. Could it make more sense to link to what ES _is_ trying to be rather than state what it _isn't_? Maybe something like "If requesting a new feature, first ask yourself: will this make the game more fun or interesting within the existing scope of the game? Keep in mind [the existing tone and vision set for Endless Sky](https://github.com/endless-sky/endless-sky/wiki/Endless-Sky's-Vision)"

Reply:
I too echo this sentiment, but I'd suggest not using that link since it *also* doesn't actually express what ES is trying to be in a particularly helpful way.